airbyte-integrations/connectors/source-amazon-ads/unit_tests/test_source.py

```diff
@@ -64,25 +65,23 @@ def test_check(config_gen):
     assert command_check(source, config_gen(start_date=...)) == AirbyteConnectionStatus(status=Status.SUCCEEDED)
     assert len(responses.calls) == 2
 
-    assert command_check(source, config_gen(start_date="")) == AirbyteConnectionStatus(status=Status.SUCCEEDED)
-    assert len(responses.calls) == 4
+    with pytest.raises(Exception):
+        command_check(source, config_gen(start_date=""))
```

Comment:
isn't start date optional?

Reply:
Yes, it is optional. But we're adding it with the wrong template in that case, that's why the error rise.